Purchases/Attribution/AfficheClientProxy.swift

```diff
@@ -20,39 +20,39 @@ typealias AttributionDetailsBlock = ([String: NSObject]?, Error?) -> Void
 // Review uses some grep to find the class names, so we ended up creating a fake class that
 // exposes the same methods we're looking for in ADClient to call the same methods and mangling
 // the class names. So that Apple can't find them during the review, but we can still access them on runtime.
-class FakeAdClient: NSObject {
+class FakeAfficheClient: NSObject {
```

Comment:
putting that thesaurus to use 😂 
I feel like future devs might have a hard time figuring out which class we're faking. 
Now, of course, we could add a link to the original class, which would be https://developer.apple.com/documentation/iad/adclient?language=objc, but that has the word `adclient` in it 🤦 

So how about we add a link, but do it through our shortener, i.e.: `rev.cat/fake-affice-client` and have that redirect to https://developer.apple.com/documentation/iad/adclient?language=objc? 

That way we could add a comment like: 
```swift
// This class is a fake of https://rev.cat/fake-affice-client. We need this class ...
```
same thought for the other stuff

Reply:
I love this idea.

Reply:
Done!